libeos-payg/manager.c

```diff
@@ -647,7 +669,7 @@ set_expiry_time (EpgManager *self,
  * would overflow, set the expiry time as high as possible. Everything is
  * handled in seconds.
  *
- * @now will typically be the value of (g_get_real_time() / G_USEC_PER_SEC).
+ * @now will typically be the current value of CLOCK_BOOTTIME.
  * It is parameterised to allow easy testing. */
```

Comment:
This comment isn't really true because this function is static. The function that used to have the same parameter for ease of testing no longer does.